packages/http-server/src/server.ts

```diff
@@ -78,12 +78,12 @@ const replyHandler = <LoaderInput>(
           reply.headers(output.headers);
         }
 
-        if (output.body) {
-          // body is already serialized
-          reply.serializer((payload: any) => payload);
-          reply.send(output.body);
-        }
+        reply.serializer((payload: unknown) => payload);
+        reply.send(output.body);
+      } else {
+        reply.code(500).send('Unable to find any decent response for the current request.');
```

Comment:
Note that in line 88 we have similar error handling. The idea was that Prism throws exceptions in such situations and Prism's clients handle those exceptions. How about throwing + handling exception instead? E.g. the error returned by Prism could have an error code we could check.

Reply:
Hmm actually looking at your implementation feel free to ignore the above comment.